pkgs/dart_tooling_mcp_server/test_fixtures/counter_app/android/app/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+plugins {
```

Comment:
general question around whether we need all of the platforms for this app? I recommend removing support for the platforms that will not be used in testing.